src/Temporalio/Workflows/WorkflowInfo.cs

```diff
@@ -32,6 +35,8 @@ public record WorkflowInfo(
         string? CronSchedule,
         TimeSpan? ExecutionTimeout,
         IReadOnlyDictionary<string, Api.Common.V1.Payload>? Headers,
+        Exception? LastFailure,
+        IReadOnlyCollection<IRawValue>? LastResult,
```

Comment:
Can you also please test last result?

Reply:
A collection of one is ok. Another benefit is that this will be empty but not null when there's no result but there was a last run.

> Can you also please test last result?

Yes, I figured it may be hard to do in a timely manner with schedules, but maybe continue as new works. I will look to see how TS tested this value.

Reply:
Added test